src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <SharedFrameworkValidationRefPath>$(RefRootPath)SharedFrameworkValidation</SharedFrameworkValidationRefPath>
+    <SharedFrameworkValidationRuntimePath>$(BinDir)runtime/SharedFrameworkValidation</SharedFrameworkValidationRuntimePath>
+  </PropertyGroup>
+
+  <Target Name="RestorePackagesProject">
+    <PropertyGroup>
+      <_RuntimeFrameworkVersion>2.0.0-preview2-25303-01</_RuntimeFrameworkVersion>
```

Comment:
yeah, I think the idea is that this will eventually live in dependencies.props so that it is automatically updated, and then this project will read it's value from there.